ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -127,6 +127,7 @@ da_scala_binary(
     deps = [
         ":ledger-api-test-tool-lib",
         ":ledger-api-test-tool-tests",
+        "//daml-lf/data",
```

Comment:
Same as above: because I want to have access to `test.shortIdentifier` in the `--list` implementation, and it is a `lf.data.Ref.LedgerString`, which is apparently provided by that module. Without it scalac complained the Ref class was not on the classpath.

Again, happy to change this if you can point me in a better direction. :-)